test/timeouts.js

```diff
@@ -132,3 +218,7 @@ test('block request gets cancelled before timeout', async function (t) {
 
   await close
 })
+
+function afterTicks (cb) {
+  setImmediate(() => setTimeout(cb, 1))
```

Comment:
why not just setImmediate?

Reply:
Ahg, it's a bit difficult to explain, it's like the reason is reversed or something haha

Consider this test:
```js
test('core constructor timeout but disable on get', async function (t) {
  t.plan(1)

  const core = await create({ timeout: 1 })

  setImmediate(() => core.append('hi'))
  t.alike(await core.get(0, { timeout: 0 }), b4a.from('hi'))
})
```

Let's say there was a bug in `{ timeout: 0 }`, so timeout is still 1

So practically the test becomes this:
```js
test('core constructor timeout but disable on get', async function (t) {
  t.plan(1)

  const core = await create({ timeout: 1 })

  setImmediate(() => core.append('hi'))
  t.alike(await core.get(0, { timeout: 1 }), b4a.from('hi'))
})
```

This test will now sometimes work, and sometimes throw

Reply:
So `afterTicks` is only used when we want to ensure that we past the 1ms timeout

Reply:
To summarize, we want this to consistently throw:
```js
setImmediate(() => core.append('hi'))
t.alike(await core.get(0, { timeout: 1 }), b4a.from('hi'))
```

Using this has same effect as previous one (sometimes works, and sometimes throws):
```js
setTimeout(() => core.append('hi'), 2)
t.alike(await core.get(0, { timeout: 1 }), b4a.from('hi'))
```

Reply:
👍 Needs a comment in afterTicks tldr’ing that